### PR TITLE
change solution structure

### PR DESCRIPTION
**Pull request by SolovyovDmitry**

### CODE REVIEW

**Comment by SolovyovDmitry** [comment]
I'd suggest pulling this branch to check final structure

File: src/OfficeService.Business/WorkspaceType/Find/FindWorkspaceTypesHandler.cs [hunk 1]
```diff
@@ -66,10 +66,10 @@ public WorkspaceTypeInfo Map(DbWorkspaceType workspaceType)
     #endregion
 
     public FindWorkspaceTypesHandler(
-      IDataProvider provider,
+      OfficeServiceDbContext dbContext,
       IBaseFindFilterValidator baseFindValidator)
```
Comment (vorobevamv) [inline src/OfficeService.Business/WorkspaceType/Find/FindWorkspaceTypesHandler.cs]: update Kernel version and remove validation with IBaseFindFilterValidator

File: src/OfficeService.Business/Office/Find/FindOfficesHandler.cs [hunk 2]
```diff
@@ -6,16 +6,16 @@
...
 using Microsoft.EntityFrameworkCore;
 
-namespace LT.DigitalOffice.OfficeService.Business.Commands.Office.Find
+namespace LT.DigitalOffice.OfficeService.Business.Office.Find
 {
   public class FindOfficesHandler : IRequestHandler<OfficeFindFilter, FindResult<OfficeInfo>>
   {
-    private readonly IDataProvider _provider;
+    private readonly OfficeServiceDbContext _dbContext;
     private readonly IBaseFindFilterValidator _baseFindValidator;
```
Comment (vorobevamv) [inline src/OfficeService.Business/Office/Find/FindOfficesHandler.cs]: don't need it

File: src/OfficeService.Business/Workspace/Find/FindWorkspacesHandler.cs [hunk 3]
```diff
@@ -6,19 +6,18 @@
...
+using LT.DigitalOffice.OfficeService.DataLayer.Models;
 using MediatR;
 using Microsoft.EntityFrameworkCore;
 
-namespace LT.DigitalOffice.OfficeService.Business.Commands.Workspace.Find
+namespace LT.DigitalOffice.OfficeService.Business.Workspace.Find
 {
   public class FindWorkspacesHandler : IRequestHandler<WorkspaceFindFilter, FindResult<WorkspaceInfo>>
   {
     private readonly IBaseFindFilterValidator _baseFindValidator;
```
Comment (vorobevamv) [inline src/OfficeService.Business/Workspace/Find/FindWorkspacesHandler.cs]: don't need it

File: src/OfficeService.Business/WorkspaceType/Find/FindWorkspaceTypesHandler.cs [hunk 4]
```diff
@@ -6,18 +6,18 @@
...
 using Microsoft.EntityFrameworkCore;
 
-namespace LT.DigitalOffice.OfficeService.Business.Commands.WorkspaceType.Find
+namespace LT.DigitalOffice.OfficeService.Business.WorkspaceType.Find
 {
   public class FindWorkspaceTypesHandler : IRequestHandler<WorkspaceTypeFindFilter, FindResult<WorkspaceTypeInfo>>
   {
-    private readonly IDataProvider _provider;
+    private readonly OfficeServiceDbContext _dbContext;
     private readonly IBaseFindFilterValidator _baseFindValidator;
```
Comment (vorobevamv) [inline src/OfficeService.Business/WorkspaceType/Find/FindWorkspaceTypesHandler.cs]: don't need it

**Review by vorobevamv — APPROVED** [review]

**Review by ndoriomedov — APPROVED** [review]

**Pull request merged**